packages/terra-date-picker/src/DatePicker.module.scss

```diff
@@ -1,90 +1,185 @@
-/* stylelint-disable selector-max-compound-selectors, max-nesting-depth */
+/* stylelint-disable max-nesting-depth */
```

Comment:
Instead of disabling for whole file, it should be only for the particular line which requires it.

Reply:
This is no longer required, so i have removed it here : https://github.com/cerner/terra-framework/pull/1208/commits/80162e0e8172a99f65f5de3b9813ac2f3b8bede5